doc/datasetManager.md

```diff
@@ -274,37 +274,104 @@ webApp:
 
 </details>
 
-## Dataset creation scripts
+### Category details
 
-### Default transformation scripts
```

Comment:
Your commit deletes the whole section _Dataset creation scripts / Default transformation scripts_, could you restore it please? :smile: